Example_App/Example - optimove.json

```diff
@@ -1,4 +1,5 @@
 {
     "optimoveCredentials": "YOUR_OPTIMOVE_CREDENTIALS",
-    "optimoveMobileCredentials": "YOUR_OPTIMOVE_MOBILE_CREDENTIALS"
+    "optimoveMobileCredentials": "YOUR_OPTIMOVE_MOBILE_CREDENTIALS",
+    "optimoveInAppConsentStrategy" : "YOUR_IN_APP_CONSENT_STRATEGY"
```

Comment:
I think for naming we can just have a key like `in-app-consent-strategy` or something.

Can we compare with Flutter / align at some point? (maybe alignment can wait to change iOS + Android together)

Reply:
We could also perhaps set the value to `auto-enroll` by default for the example.

Hm, are we using `enrol` or `enroll` spellings...

Reply:
(actually maybe disabled is better)

Reply:
sure I'll look at the flutter sdk